### PR TITLE
Implement slack messaging to multiple channels.

### PR DESCRIPTION
**Pull request by seancorc**
**- Git Issue:**: #8 

**- Context:** Before these changes, if a test case failed the integration slack app used a webhook to post all of the test results into the "#integration-results" channel.

**- Changes:** 
1. Slack:

-  Created new slack channels for each application being tested.

-  Created a new slack app 'Integration-v2' which can post in these new slack channels via webhooks.

2. Code: 

- Add SlackMessage class and add slack_message field to TestGroup class.

-  Add dicts to keep track of each application's webhook url and user ids.

-  Individualize the slack text for each testgroup.

### CODE REVIEW

**Comment by seancorc** [comment]
Note that the PR test will fail as long as the server environment variables do not match.

File: src/main.py [hunk 1]
```diff
@@ -18,50 +17,46 @@
 SUCCESS_STATUS = "SUCCESS"
 FAILURE_STATUS = "FAILURE"
 
-num_tests = sum([len(test_group.tests) for test_group in test_groups])
-num_failures = 0
-application_error_tracking = {Application.EATERY: False, Application.TRANSIT: False, Application.UPLIFT: False}
-
-slack_message_text = "*Starting new test run...*\n"
+application_userID_mapping = {
```
Comment (meganle) [inline src/main.py]: let's stay consistent with snake case and make it `application_user_id_mapping`

**Review by kq-li — APPROVED** [review]
sick, let's ship and try it out

**Review by meganle — APPROVED** [review]
nice work!!

**Review by manishsaha — APPROVED** [review]

**Pull request merged**